modules/govuk/templates/node/s_graphite/remove_old_whisper_data.erb

```diff
@@ -14,6 +14,14 @@ for dir in $whisper_dir/*; do
     rm -rf $dir;
   fi
 done
+
+for dir in stats stats_counts; do
+  find "$whisper_dir/$dir" -name "*ip-*" -type d -print0 | while IFS= read -r -d '' dir; do
```

Comment:
Could we do `".ip-*"` here? I'm thinking other metrics may contain a `'*ip-*'` string somewhere.

Reply:
Some of the metrics that this change looks at deleting don't necessarily start with `ip-`, like `/opt/graphite/storage/whisper/stats_counts/calculators_frontend-ip-10-12-6-209_eu-west-1_compute_internal` for example.